uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_leave-calendar.scss

```diff
@@ -164,21 +164,27 @@ $chr-leave-calendar-legend-size:                  20px;
   }
 
   > .chr_leave-calendar__item {
-    float: left;
     font-size: 8px;
     height: $chr-leave-calendar-legend-size;
     margin-right: 10px;
+    position: absolute;
```

Comment:
Could you please explain what it fixes and how?

Reply:
It fixes the legend, where the 3rd line used to be shifted to the left. But after removing `float: left;`, that does not happen anymore. But still to keep the text adjacent to the Icon, it is necessary to add `position: absolute;`, so that its layout is calculated separately, and it does not interfere with the text's position. Does that make sense?